lib/keen/UiToolbar.vue

```diff
@@ -116,6 +118,14 @@
     },
 
     methods: {
+      getColor() {
+        if (this.textColor === 'default') {
+            return this.$themeTokens.surface;
+        } else {
+        const tokens = this.textColor.split('.');
+        return this.$themeTokens[tokens[1]] || '';
```

Comment:
This is for passing any of the theme tokens listed in the KDS documentation. For example, in the above-given screenshot, I used the themetoken.document token, and it worked.
![Screenshot from 2024-01-26 19-05-54](https://github.com/learningequality/kolibri-design-system/assets/77477551/ce42f3f9-548b-44f9-a451-91f1cb430cd4)

Reply:
`textColor="$themeTokens.document"` means that you're passing string `"$themeTokens.document"` rather than variable `$themeTokens.document` that will evaluate into a color code.

If we were going to pass theme tokens via the `textColor` prop, then we'd want to accept them like `:textColor=$themeTokens.document` which is a shortcut for `v-bind` that passes variables https://v2.vuejs.org/v2/guide/components-props#Passing-Static-or-Dynamic-Props. Then you wouldn't need

```
const tokens = this.textColor.
return this.$themeTokens[tokens[1]] || '';
```

However, looking at the approach you took overall, I'd recommend you continue accepting only `'black'` and `'white'` as `textColor` and translated them into theme colors as previously explained here https://github.com/learningequality/kolibri-design-system/pull/536#discussion_r1476049877.